lms/djangoapps/certificates/tests/test_queue.py

```diff
@@ -73,6 +78,58 @@ def test_no_create_action_in_queue_for_html_view_certs(self):
         self.assertEqual(certificate.status, CertificateStatuses.downloadable)
         self.assertIsNotNone(certificate.verify_uuid)
 
+    @ddt.data(
+        ('honor', 'honor'),
+        ('audit', 'audit')
+    )
+    @ddt.unpack
+    def test_add_cert_with_honor_certificates(self, mode, expected_mode):
+        """Test certificates generations for honor and audit modes."""
+        template_name = 'certificate-template-{id.org}-{id.course}.pdf'.format(
+            id=self.course.id
+        )
+        self.assert_queue_response(mode, expected_mode, template_name)
+
+    @ddt.data(
+        ('credit', 'verified'),
```

Comment:
Again, you only need a single list of strings since the expected mode is always verified.